.github/workflows/build.yml

```diff
@@ -156,6 +156,10 @@ jobs:
           - component: ssh-bastion
             context: ./operators/build/ssh-bastion
 
+          # img-exporter image for InstanceSnapshot
```

Comment:
@giorio94 shouldn't there be also a line with `img-exporter` before, e.g. at line 78 of this file?

Reply:
Definitely. Thanks, I missed that.